pkgs/racket-doc/scribblings/reference/reader.scrbl

```diff
@@ -956,12 +956,16 @@ numbers are followed by a @litchar{.} intended to be read as a C-style
 infix dot, then there must be separating whitespace.
 
 Finally, after reading any value, @racket[_x], the reader will seek
-over whitespace until it reaches a non-whitespace character. If the
-character is not @litchar{.}, then the value, @racket[_x], is returned
-as usual. If the character is @litchar{.}, then another value,
-@racket[_y], is read and the result @racket[(list '#%dot _x _y)] is
-returned. In @racket[read-syntax] mode, the @racket['#%dot] symbol has
-the source location information of the @litchar{.} character and the
+through whitespace and look for zero or more sequences of a
+@litchar{.} followed by another value @racket[_y], to put each
+@racket[_y] together with @racket[_x] in a @racket[#%dot] form.
+@racket[_x.y] will be read as @racket[(#%dot _x _y)].
+
+It does this in a left-associative way so that @racket[_x.y.z]
+will be read as @racket[(#%dot (#%dot _x _y) _z)].
```

Comment:
It's not really that `y` contains a `.`, it's that there's another `.` after `y`. So would "If `y` has another `.` after it, the reader will group `.`-separated datums from left-to-right" be better?